homeassistant/components/huawei_lte/sensor.py

```diff
@@ -183,7 +215,7 @@ class HuaweiSensorEntityDescription(SensorEntityDescription):
             ),
             "ltedlfreq": HuaweiSensorEntityDescription(
                 key="ltedlfreq",
-                name="Downlink frequency",
+                name="LTE downlink frequency",
                 formatter=lambda x: (
```

Comment:
Note: in #83904 I noticed there were quite a few multi-line lambdas.
Is there a way they could be converted to single line lambdas or moved to stand-alone functions?

Reply:
Sure, at least named functions. I'm not sure multiline lambdas are a problem per se. c530a0e72a93f273748831a0ac644b9669c6ba67 converts bodies of ones that are used more than once to named functions; converting ones that are used only does not seem to be a net positive to me.

Reply:
@epenet made the requested changes, any chance for a review here? :)